gh-update-branch

```diff
@@ -1,8 +1,22 @@
 #!/bin/bash
 set -e
 
-read -r number sha < <(gh pr view ${1} --json number,commits --jq '[.number,.commits[-1].oid]|@tsv')
+if [ -z "$1" ]; then
+  prs=$(gh pr view --jq '.[].number' --json number)
+  echo "Updating current PR"
+elif [ $1 == "--all" ]; then
+  prs=$(gh pr list --author @me --jq '.[].number' --json number)
```

Comment:
One of the things adding this doesn't allow for is updating PRs *not* by you in quick unison, which is actually why I wrote this back when I had to use the tool, as a reviewer might want to update several PRs by someone else in quick succession. 

I wonder if maybe instead of `--all`, `--mine` would be a better name? Or perhaps having a `--author` that defaults to `@me` 🤔

Reply:
Good point. I think `--mine` should be the name of the option.
Also if this will one day be upstreamed to GitHub CLI it is useless to complicate the logic flow for nothing.
Personally I don't update branches that are not mine at the moment, so I'm not really interested in supporting that pattern